var/spack/repos/builtin/packages/fortrilinos/package.py

```diff
@@ -72,3 +74,28 @@ def cmake_args(self):
             self.define('ForTrilinos_EXAMPLES', self.run_tests),
             self.define('ForTrilinos_TESTING', self.run_tests),
         ]
+
+    examples_src_dir = 'example/test-installation'
+
+    @run_after('install')
+    def setup_smoke_tests(self):
+        """Copy the example source files after the package is installed to an
+        install test subdirectory for use during `spack test run`."""
+        self.cache_extra_test_sources([self.examples_src_dir])
+
+    def test(self):
+        test_build_dir = join_path(self.install_test_root,
+                                   self.examples_src_dir, 'build')
+        mkdirp(test_build_dir)
+        with working_dir(test_build_dir):
```

Comment:
```suggestion
        with working_dir(test_build_dir, create=True):
```

Reply:
I copied this from arborx, so @masterleinad might want to do the same there :)